code/__DEFINES/components.dm

```diff
@@ -404,3 +404,8 @@
 
 //Heretics stuff
 #define COMSIG_HUMAN_VOID_MASK_ACT "void_mask_act"
+
+//NSV13 comsigs begin
+#define COMSIG_GLOB_CHECK_INTERDICT "check_interdict"							//from interdiction component
+#define BEING_INTERDICTED (1<<0)											//returned on successful interdict
```

Comment:
Why not just make this 1?

Reply:
Future-proofing to allow easyish adding of different interdict behavior via additional bitfields which would not be possible via only a normal number returned.

Reply:
Agree with the use of bitflags here.